examples/cross_sign_verify.py

```
@@ -0,0 +1,129 @@
+        HOMESERVER, USER_ID, store_path=STORE_FOLDER, config=config
+    )
+
+    await client.login()
+
+    print("Syncing with the server...")
+    await client.sync()
+
+    print("Verifying own device...")
+    response = await client.verify_own_device(PRIVATE_SELF_SIGNING_KEY)
+
+    if isinstance(response, KeysSignaturesUploadResponse):
```

Ruff (0.12.2)

Undefined name `KeysSignaturesUploadResponse`

(F821)